deploy/rbac-permissions-operator-config/05-dedicated-admins-aggregate-cluster.ClusterRole.yaml

```
@@ -270,3 +270,12 @@ rules:
   verbs:
   - use
 ### END
+### Start - manage project.config.openshift.io CR
+# https://issues.redhat.com/browse/OSD-5566
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - "*"
```

```suggestion
  - get
  - list
  - patch
  - update
  - watch
```
Probably better to not allow delete, and therefore not allow create.